template/default.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "__alertmanager" }}AlertManager{{ end }}
-{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}{{ end }}
```

Comment:
Ok, so to clarify: does anything need to be done, or is the regexp escaping on the front end when creating the URLs sufficient?